app/main.py

```diff
@@ -82,7 +84,3 @@ def metrics(segments):
         'numEdges': G.number_of_edges(),
         'nodes': nodes
     }
-
-
-if __name__ == '__main__':
-    metrics.interface.cli()
```

Comment:
Did you use this feature of hug often? To use the API as a CLI?

Reply:
No, we did not use this at all.

Reply:
Alright.